style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  width: 320px;
+}
```

Comment:
- [ ] setting a fixed width is creating an unnecessary space on your site better to use flexible units, Kindly style the section so that it looks similar to the design.

Your Implementation | Figma Design
:---: | :---:
![image](https://user-images.githubusercontent.com/59416136/153298750-688cde56-2562-45bd-ad38-1513cada8874.png) | ![image](https://user-images.githubusercontent.com/59416136/153298824-cd824237-680a-4374-97aa-324365ac4090.png)